app/jobs/csv_export_job.rb

```diff
@@ -34,22 +34,20 @@ def deploy_csv_export(csv_export)
     filename = csv_export.path_file
     filter = csv_export.filters
 
-    @deploys = Deploy.joins(:stage, :job).where(filter)
+    @deploys = Deploy.joins(
+      'INNER JOIN stages ON stages.id = deploys.stage_id INNER JOIN jobs ON jobs.id = deploys.job_id'
+    ).unscope(where: :deleted_at).where(filter)
```

Comment:
This works the same as unscoped, but instead of removing the entire default_scope it just removes from the query "WHERE `deleted_at` IS NULL" condition.  This was when I was trying to join with the associations using joins(:jobs, :stages) but fails since this only applies on the main Model and not associations.